src/__tests__/__mocks__/__featureMocks__/propsMock.js

```diff
@@ -0,0 +1,166 @@
+import {
+  nextSlide,
+  previousSlide,
+  setInitData,
+} from '../../../actions/featureActions';
+
+const props = {
+  data: {
+    artList: [
+        {
+          objectID: 1,
+          url: '../assets/images/1.jpg',
```

Comment:
You need to update these guys as discussed

Reply:
done

Reply:
The url is still a path...